docs/user-guide/certificates-configuration-questionnaire.md

```diff
@@ -0,0 +1,116 @@
+# Zowe certificates questionnaire
+
+To properly configure Zowe to use certificates for server-side component installation, it is important to understand certificate setup options. Understanding these options makes it possible to select the best [certificate configuration scenario](./certificate-configuration-scenarios.md) that fits your Zowe deployment use case. To assist you in making this certificate scenario selection, it is useful to familiarize yourself with the Zowe certificate decision process, and answer the questions in the questionnaire at the end of this article.
+
+**Tip:**
+Before answering the questionnaire you may want to make your self familiar with the certificate configuration basics and Zowe certificates configuration overview.
+- [Certificates concepts](../appendix/zowe-security-glossary#certificate-concepts) explained in our [Zowe Security Glossary](../appendix/zowe-security-glossary)
+- [Zowe certificates overview](../getting-started/zowe-certificates-overview)
```

Comment:
I'd add that choosing a scenario is not mandatory if the user has a pre-configured keyring, they can skip the `zowe.setup.certificate` section altogether and just fill the keyring information in `zowe.certificate`

Reply:
Maybe this note will be more relevant in the configuration scenarios document. Here we just aim to navigate the user to the right scenario, asumming they need to.